engine/tests/phpunit/unit/Elgg/Application/BootHandlerUnitTest.php

```diff
@@ -48,7 +47,7 @@ public function testCanDoFullBoot() {
 
 		$app->bootCore();
 
-		$this->assertTrue($app->internal_services->config->boot_complete);
+		$this->assertTrue($app->getBootStatus('full_boot_completed'));
```

Comment:
please check the other status also (like in the tesk below this, only now `true`)